.github/workflows/bazel.yml

```diff
@@ -638,6 +648,89 @@ jobs:
           echo "Available storage:"
           df -h
 
+      # Create Sentry.io release with the services connectiond, liagentd, mme, sctpd and sessiond.
+      - run: curl --silent --location https://sentry.io/get-cli/ | bash
+        if: |
+          github.event_name == 'push' &&
```

Comment:
does it make sense to move this to a separated follow-up job that depends on the package job? i.e., like before publish the artifacts and download them in the next job
* improves separation of concerns
* reduces the boilerplate to a one-time if-check
```
github.event_name == 'push' &&
github.repository_owner == 'magma' &&
github.ref_name == 'master'
```
* could the upload steps even be modeled by a matrix construct?

Reply:
Good idea, I moved the steps to a separate job - very similar to how it was done in the build all workflow. Using a matrix job would be too complicated, because the final release step would then also require a separate job as it should only run once.